l3embedding/train.py

```diff
@@ -238,32 +238,20 @@ def sample_one_frame(video_data, start=None, fps=30, scaling_func=None, augment=
     return frame_data, frame / fps, video_aug_params
 
 
-def sampler(video_file, audio_files, augment=False):
-    """Sample one frame from video_file, with 50% chance sample one second from corresponding audio_file,
-       50% chance sample one second from another audio_file in the list of audio_files.
-
-    Args:
-        video_file: video_file to sample from
-        audio_files: candidate audio_files to sample from
-
-    Returns:
-        A generator that yields dictionary of video sample, audio sample,
-        and label (0: not from corresponding files, 1: from corresponding files)
-
-    """
-
-    video_data = vread(video_file)
-    audio_file = video_to_audio(video_file)
+def sampler(video_files, audio_files, augment=False):
+    while True:
+        video_file = random.choice(video_files)
```

Comment:
With this way we would be opening a new video file each time, which could be slow on IO